<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ai_data_import</class>
 <widget class="QWidget" name="ai_data_import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三元组数据处理</string>
  </property>
  <widget class="QPushButton" name="select_config_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>选择配置文件</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2、数据类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3、数据所在目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_data_folder">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>选择数据所在目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exec_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>4、添加数据到指定路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1、选择三元组配置文件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>数据存放目录</string>
   </property>
  </widget>
  <widget class="QLabel" name="target_data_folder">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
